src/api/spec/models/branch_package_spec.rb

```diff
@@ -97,5 +97,19 @@
         end
       end
     end
+
+    context 'retains url' do
+      it 'for a newly created package' do
+        branch_apache_package.branch
+        package = Package.where(name: 'apache2').where.not(id: apache.id).first
+        expect(package.url).to eq(apache.url)
```

Comment:
Somehow this spec is now failing? Were some of the changes lost in the rebase?

Reply:
apologies, but it should be resolved now.

Reply:
No need to apologize, mistakes happen :slightly_smiling_face: